tests/js/JSONFormatBearTest.py

```diff
@@ -91,3 +94,17 @@ def test_exception_empty_file(self):
                                               invalid_files=(),
                                               settings={'escape_unicode':
                                                         'false'})
+
+
+JSONFormatBearInfLineLengthTest = verify_local_bear(JSONFormatBear,
+                                                    valid_files=(
+                                                        max_line_length_file,),
+                                                    invalid_files=())
+
+
+JSONFormatBearLineLengthTest = verify_local_bear(JSONFormatBear,
+                                                 valid_files=(test_file1,),
+                                                 invalid_files=(
+                                                    max_line_length_file,),
```

Comment:
four space indentation.

and rebase!

(but wait until you get a green build, as you need to check you reached 100% test coverage)

Reply:
Damn, how did I miss that.